pkg/services/services.go

```diff
@@ -330,6 +339,22 @@ func (s *WalletServices) IsUtxo(ctx context.Context, scriptHash string, outpoint
 	return result, nil
 }
 
+// GetStatusForTxIDs returns depth/status info for a list of txIDs.
+func (s *WalletServices) GetStatusForTxIDs(ctx context.Context, txIDs []string) (*wdk.GetStatusForTxIDsResult, error) {
+	if len(txIDs) == 0 {
+		return nil, fmt.Errorf("no txIDs provided")
+	}
+
+	res, err := s.getStatusForTxIDsServices.OneByOne(ctx, txIDs)
+	if err != nil {
+		if errors.Is(err, servicequeue.ErrEmptyResult) {
+			return nil, fmt.Errorf("no status found for provided txIDs")
```

Comment:
It's sometimes necessary to check if there was "not found" situation.
```suggestion
			return nil, fmt.Errorf("no status found for provided txIDs: %w", wdk.NotFoundError)
```

For this, general-purpose `wdk.NotFoundError` was created.

This is also applicable to other functions. Could you do this as your next task?